### DISCUSSION TITLE
Reviewer

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1063] [Translation] [answered]
Hello, I want to be reviewer, how can i be.
I'm translating the scratch addons into Turkish
My transifex username is **Translator20**

**Comment by GrahamSH-LLK** [comment, marked as answer]
Hi! We keep reviewers separate when we can in order to have the highest quality of translation. Reviewers check over your translations to make sure we are providing accurate translations to our users. @ahmetlii is the reviewer for Turkish, and you can ask him any questions you have. He will review translations once they are complete.

**Reply by Thomas282-coder** [reply]
ok, thanks :)
